openpype/hosts/zbrush/api/__init__.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+"""Public API for Zbrush"""
+from .communication_server import CommunicationWrapper
+from .pipeline import (
+    ZbrushHost
```

Comment:
Is `ZbrushHost` still undefined? Pretty sure this will error currently. :)

Reply:
As it doesn't get used anywhere yet, so it doesn't get any error so far.